app/views/devise/registrations/new.html.erb

```diff
@@ -4,38 +4,38 @@
   <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
     <%= render "devise/shared/error_messages", resource: resource %>
     <div class="field mt-4">
-      <%= f.label :full_name, class: "block text-gray-700" %>
+      <%= f.label :full_name, class: 'block text-gray-700' %>
```

Comment:
Why change to single quote in this file?

Reply:
good one! fixed